Api/src/routes/officeRoutes.js

```diff
@@ -0,0 +1,40 @@
+import express from 'express';
+
+import {
+  getOffices,
+  createOffice,
+  getOffice,
+  deleteOffice,
+  updateOffice,
+} from '../controllers/officeController.js';
+
+const officeRouter = (app) => {
+  app.get('/offices', getOffices)
+  app.post('/offices', createOffice)
+  app.get('/offices/:id', getOffice)
+  app.patch('/offices/:id', updateOffice)
+  app.delete('/office/:id', deleteOffice)
+}
+
+export default officeRouter;
+
+
+// const officeRouter = express.Router();
+
+// officeRouter.route("/")
```

Comment:
Like I said above, this is dirty and we should not ask for a review with stuff like this in the code. And self-review should catch and remove these.

Reply:
Cleanup